### DISCUSSION TITLE
Better support/scaling for high DPI screens

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#233] [Feature requests]
As seen in my screenshot here:
![unknown](https://user-images.githubusercontent.com/25685344/193442045-ba785c9b-dff4-4111-8c1f-6edd89e60574.png)

Even on a 27" 1440p screen the live timing stuff doesn't seem to scale too well on high DPI screens. Some of the text is too small or it doesn't correctly shift into a two-column setup in the case of the live timing.

Possibly making adjustments to @media queries could help https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries

Another possibility to enable the live timing to be compressed down further is to be able to toggle which elements are shown. for example, turning off speed info or mini sectors. And making some other elements smaller when the window is made smaller.

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #194
